cpu/cpu_test.go

```diff
@@ -110,6 +111,13 @@ func loadMockCPUInfo(dataSetNumber int) {
 		content = `cpu 180401494 227200 18747745 3823269793 1561918 12082 2511349 0
 			cpu0 22541572 28113 2329501 477843628 173611 1735 315175 0
 			cpu1 23343161 22869 2630545 476714355 160618 1759 329698 0`
+	} else if dataSetNumber == twoDigitCPUIdentifierStatIndex {
+		content = `cpu 180401494 227200 18747745 3823269793 1561918 12082 2511349 0 0 0
+			cpu0 3464284 998669 208226 49355234 57380 3 422 0 0 0
+			cpu1 3501681 1012206 189642 49374240 11620 0 278 0 0 0
+			cpu10 22541572 28113 2329501 477843628 173611 1735 315175 0 0 0
+			cpu11 23343161 22869 2630545 476714355 160618 1759 329698 0 0 0
+			intr 33594809 19 2 0 0 0 0 0 9 1 4 0 0 4 0 0 0 31 0 0`
```

Comment:
Consider adding cpu10, cpu11 and so on to standard content and do not distinguish `twoDigitCPU` case that in that way. Just make one content, but more representative and run all tests for that one.